table/tables/tables.go

```diff
@@ -1592,6 +1595,10 @@ func shouldWriteBinlog(ctx sessionctx.Context, tblInfo *model.TableInfo) bool {
 	return !ctx.GetSessionVars().InRestrictedSQL
 }
 
+func shouldIncreaseTTLCount(tblInfo *model.TableInfo) bool {
+	return tblInfo.TTLInfo != nil && tblInfo.TTLInfo.Enable
```

Comment:
I think we only need to check `tblInfo.TTLInfo != nil` ?

Reply:
You're right.